docs/overview.yaml

```diff
@@ -282,6 +282,7 @@ Analysis:
     derivative of the inverse of a function: 'has_fderiv_at.of_local_left_inverse'
     Rolle's theorem: 'exists_deriv_eq_zero'
     mean value theorem: 'exists_ratio_deriv_eq_ratio_slope'
+    Taylor's theorem: 'analysis/calculus/taylor.html'
```

Comment:
references in this file refer to some theorem name rather than html files, I think.

Reply:
The liminf/limsup entry is also a html file and I thought that it might be better if we only have one entry for all the variants of Taylor's theorem. If you think that it is better to have all versions separately, I can certainly change it.

Reply:
I agree it makes sense to keep a reference to the whole file here.